File: 1501-1750/1512. Number of Good Pairs/solution.dart
class Solution {
  int numIdenticalPairs(List<int> nums) {
    var n = nums.length ; 
    var rs = 0 ; 
    for( int i=0 ; i<n ; ++i ){
        for(int j=i+1 ; j<n ; ++j){
            if( nums[i] == nums[j] ){
                rs++ ; 
            }
        }
    }
    return rs ; 

  }
}

File: 1501-1750/1512. Number of Good Pairs/solution.py
class Solution(object):
    def numIdenticalPairs(self, nums):
        rs = 0 
        for i in range (len(nums)) :
            for j in range( i+1 ,  len(nums) ): 
               if nums[i] == nums[j] :
                rs += 1 
        return rs
        